Gruntfile.js

```diff
@@ -133,43 +133,36 @@ module.exports = function( grunt ) {
 
 					"requirejs/require.js": "requirejs/require.js",
 					"requirejs-domready/domReady.js": "requirejs-domready/domReady.js",
-					"requirejs-text/text.js": "requirejs-text/text.js",
+					"requirejs-text/text.js": "requirejs-text/text.js"
 				}
 			}
 		},
-		jshint: {
+		eslint: {
 			options: {
-				jshintrc: true
+
+				// See https://github.com/sindresorhus/grunt-eslint/issues/119
+				quiet: true,
+				configFile: ".eslintrc",
+				fix: true
 			},
-			all: {
-				src: [ files.source, files.grunt, files.karma, files.speed, files.tests ]
-			}
-		},
-		jscs: {
+
+			// We have to explicitly declare "src" property otherwise "newer"
+			// task wouldn't work properly :/
 			src: {
-				options: {
-					requireDotNotation: null
-				},
-				src: [ files.source ]
+				src: files.source // requireDotNotation: null
```

Comment:
Is there any reason this needs to exist separately instead of merging `src` & `dev` into `all`?